### ISSUE TITLE
Downloading old Live Streams from Live tab NOT Videos with channel starting @

### ISSUE THREAD

**Issue by MrGingerT** [meeb/tubesync#322]
Hello All,

I have beenm using Tubesync great for over a year and 1 channel I subscrieb to "Live on Patrol"   https://www.youtube.com/@liveonpatrol  https://www.youtube.com/channel/UCkHBaplUW2OH3_jqltpOK8g  has moved all of its videos (old live streams) under the tab Live. Tubesync does not seem to detect them but does download 3 videos from the Videos tab.

Whether is makes or difference on not I have to add the source using the add source by Youtube channel ID as it begins with a " @ "

Is there any way of asking Tubesync to use the Live Tab ? (There are no playlists either). 

I do not believe it a bug but more a feature request ? but again not sure
Running tubsync on unraid community app (which is a docker) - V12 running

Thank you in advance for any help

**Comment by meeb** [comment]
Yeah, YouTube has moved some content to different tabs which no longer makes all the videos appear under `/videos` for whatever reason that I'm sure makes sense to someone at YouTube. This will need a code fix so live videos only under the live tab won't download for the moment.

**Comment by thecreativeone91** [comment]
I’m starting to run into this a lot as well.

**Comment by artkrz** [comment]
I would really like this feature as well.

**Comment by mikeweyandt** [comment]
Have a pull request that would add stream indexing and downloading: https://github.com/meeb/tubesync/pull/453

Couldn't get it to work for playlists though currently, but it does work for channel/channel ids

**Comment by tcely** [comment]
It looks like this was completed after #523 was merged.

**Comment by meeb** [comment]
Yep, this looks resolved. I'll close it for now. Please open a new issue if you continue to experience issues.
